cmd/kube-proxy/app/server.go

```diff
@@ -335,12 +335,13 @@ func (s *ProxyServer) Run() error {
 }
 
 func getConntrackMax(config *options.ProxyServerConfig) (int, error) {
-	if config.ConntrackMax > 0 && config.ConntrackMaxPerCore > 0 {
-		return -1, fmt.Errorf("invalid config: ConntrackMax and ConntrackMaxPerCore are mutually exclusive")
-	}
 	if config.ConntrackMax > 0 {
+		if config.ConntrackMaxPerCore > 0 {
+			return -1, fmt.Errorf("invalid config: ConntrackMax and ConntrackMaxPerCore are mutually exclusive")
+		}
 		return int(config.ConntrackMax), nil
-	} else if config.ConntrackMaxPerCore > 0 {
+	}
+	if config.ConntrackMaxPerCore > 0 {
```

Comment:
It' s according to the go style guide advises,  when  the last "if" return,   it needn't "else".